modules/odc-removing-services-to-application.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
+
+[id="odc-removing-services-to-your-application_{context}"]
+= Removing services to your application
```

Comment:
You can extend the suggestions you choose to apply from the previous topic to this one.

Reply:
```suggestion
= Removing services from your application
```